### PR TITLE
OSDOCS-9813: adds custom CA relnote MicroShift

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.16

Issue:
[OSDOCS-9813](https://issues.redhat.com/browse/OSDOCS-9813)

Link to docs preview:
[Custom CAs release note](https://75373--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-16-release-notes.html#microshift-4-16-custom-cert-auths)

QE review:
- [x] QE has approved this change.

Additional information:
[Feature work docs](https://github.com/openshift/openshift-docs/pull/74615)

### CODE REVIEW

File: microshift_release_notes/microshift-4-16-release-notes.adoc [hunk 1]
```diff
@@ -58,15 +58,19 @@ See the following list for details:
 //[id="microshift-4-16-installation"]
 //=== Installation
 
-//[id="microshift-4-16-configuring"]
-//===Configuring
+[id="microshift-4-16-configuring"]
+=== Configuring
+
+[id="microshift-4-16-custom-cert-auths"]
+==== Customizable certificate authorities now supported
```
Comment (eslutsky) [inline microshift_release_notes/microshift-4-16-release-notes.adoc:65]: we should mention that the certificate are for the  API server in the title

**Comment by eslutsky** [comment]
/lgtm
looks good

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Pull request merged**